migrations/versions/d975ea83b712_add_seft_ci_file_location.py

```diff
@@ -0,0 +1,25 @@
+"""add seft ci file location column
+
+Revision ID: d975ea83b712
+Revises: ccd22863e633
+Create Date: 2021-09-06 10:40:27.089307
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.types import Integer, String
+
+# revision identifiers, used by Alembic.
+revision = "d975ea83b712"
```

Comment:
where did this id come from?  were they copied from somewhere else or did you have to create it?